### ISSUE TITLE
can't delete countdown timer

### ISSUE THREAD

**Issue by tboudri** [ralfvd/nl.bevlogenheid.countdown#24]
When i click on the delete button next to a countdown timer in settings screen, the timer seems to be deleted but when i leave the settings page and the come back the countdown timer is still there

**Comment by ralfvd** [comment]
Which version of countdown and which version of Homey are you running?

When I delete via the mobile app, it is deleted and after going back to the settings page it remains gone. So not sure what is happening in your case, although I’ve heard it also from other users. 

Can you show me some screenshots?

**Comment by tboudri** [comment]
I am working in the desktop app (1.5.13) And there it is not working.
If i delete a timer it disappears from the list but after refresh of the page the countdown is still there.

No message, no errors etc

I'am using version 0.9.12. I can't upgrade because the newer version needs Homey V2


Best regards

Tjeerd Boudri

**Comment by Moskvitch** [comment]
Hi, same exact problem here
I am on Homey 2 official, Countdown v 2.0.3
Deleting a timer makes it disappear from the screen, after leaving the config page and coming back it’s still on the list.

**Comment by Moskvitch** [comment]
Ok, just thought of trying something and it worked!
If you EDIT the value of the timer to be deleted from -1 to something like a 100 (give yourself enough time), then SAVE, leave the page, come back, see it counting down and then DELETE again, then it is deleted.
Apparently the app will not murder timers that pretend to be dead by not making the tecking sound :)
BTW, a great app, use it all the time(r)! 👍

**Comment by edwinvdpol** [comment]
Thanks @Moskvitch, that worked for me! You cannot delete timers which are 'dead' in Homey v2 (mobile app).

**Comment by tboudri** [comment]
@Moskvitch, Thanks. This worked for me on Homey 1.5.13 and app version 0.9.12

**Comment by marcus-karlsson** [comment]
@ralfvd @Moskvitch 
Same here cannot delete expired timer. 
Can we have a fix for it?

**Comment by satbopper** [comment]
still no fix for not being able to remove a timer,is countdown still under developing or can we forget it

**Comment by satbopper** [comment]
the workaround did delete it

**Comment by MP-ICT-Diensten** [comment]
Indeed, timers cannot be deleted. Please solve issue (I have not the skills to do that.
issue exist on Homey version 3.2.x and also 4.0.0

The workaround does still work.

**Comment by ralfvd** [comment]
Sorry for the massive delay; this will be -hopefully- fixed in an upcoming release 2.0.4 which is now being certified by Athom.  On my side it works, so hopefully everybody is able to delete timers again, after installing 2.0.4 ; in a couple of days I hope!

**Comment by wlauvenberg** [comment]
Installed the update and I can delete a timer from the config, but only one a time. To remove multiple, I had to remove one, go out of the settings screen by using the back button, open the settings screen again and delete the next one.

**Comment by ralfvd** [comment]
Hmm that’s a use case I didn’t test. I will test and see if that can be
fixed.

Thanks for the feedback!
